raml/response_examples/200/runbook_runs.json

```diff
@@ -0,0 +1,84 @@
+{
+	"runs": [
+		{
+			"_id": "434",
+			"finished_at": "2016-12-20T16:27:42.150227",
+			"output": "SGVsbG8gV29ybGQhCg==",
+			"return_code": 0,
+			"started_at": "2016-12-20T16:18:42.150142",
```

Comment:
What you're saying is we don't need seconds and mseconds in the format? I don't think it's a good idea